#this is used for Flip Motor Angle Info
float32 FrontAngle
float32 RearAngle	

